notifications/notifications.py

```diff
@@ -49,3 +53,24 @@ def message_sender(self):
 
     def start_worker(self):
         threading.Thread(target=self.message_sender, daemon=True).start()
+
+    def play_notify_sound(self):
+        self.play(NOTIFICATION_SOUND_PATH)
+
+    def play_alarm_sound(self):
+        self.play(ALARM_SOUND_PATH)
+
+    def play_purchase_sound(self):
+        self.play(PURCHASE_SOUND_PATH)
+
+    def play(self, audio_file=None, **kwargs):
+        if self.sound_enabled:
+            try:
+                # See https://github.com/TaylorSMarks/playsound
+                playsound(audio_file if audio_file else NOTIFICATION_SOUND_PATH, True)
```

Comment:
Does this block the main thread, if so, can we set this arg to False to have it run asynchronously?

Reply:
I inherited that from the previous version.  We can set blocking to false, but that's not implemented on Linux (per the playsound documentation), so I went with what previously existed.  Blocking is a problem where we spam the sounds, but in this version, I have dramatically reduced the instances of when we play sounds and I shortened the notification.mp3 to 1.5 seconds.  I'm comfortable switching it to asynchronous, but can't verify Linux behavior.

Reply:
Giving non-blocking audio a shot.  Linux users will have to verify.